spec/carrier_account_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EasyPost::CarrierAccount do
+  before do
+    EasyPost.api_key = ENV['EASYPOST_PROD_API_KEY']
+  end
+
+  describe '.create' do
+    it 'creates a carrier account' do
+      carrier_account = described_class.create(Fixture.basic_carrier_account)
+
+      expect(carrier_account).to be_an_instance_of(described_class)
+      expect(carrier_account.id).to match('ca_')
+      expect(carrier_account.type).to eq('UpsAccount')
```

Comment:
We probably don't need this assertion here, according to PHP and Java test suites.

Reply:
This should have been added to those other tests - each create test should try asserting something from the parameters.